README.md

```diff
@@ -112,6 +109,11 @@ options it will fail with `EINVAL (Invalid argument)`.
 
 #### xbps-uchroot(1)
 
+This is the only method that implements functionality of `xbps-src -t`, therefore the
+flag enables it ignoring choice in configuration file.
+
+XBPS utility that uses `namespaces` and must be `setgid` (part of xbps).
+
```

Comment:
```suggestion
XBPS utility that uses `namespaces` and must be `setgid` (part of `xbps` package in Void Linux).

```

Maybe this should go before the "This is the only..." paragraph?

Reply:
It is part of xbps whereever it is extracted, remark is more useful when you are outside Void.